elixir/apps/api/lib/api/client/views/relay.ex

```diff
@@ -1,26 +1,24 @@
 defmodule API.Client.Views.Relay do
   alias Domain.Relays
 
-  def render_many(relays, expires_at, stun_or_turn \\ :turn) do
-    Enum.flat_map(relays, &render(&1, expires_at, stun_or_turn))
+  def render_many(relays, salt, expires_at, stun_or_turn \\ :turn) do
+    Enum.flat_map(relays, &render(&1, salt, expires_at, stun_or_turn))
   end
 
-  def render(%Relays.Relay{} = relay, expires_at, stun_or_turn) do
+  def render(%Relays.Relay{} = relay, salt, expires_at, stun_or_turn) do
     [
-      maybe_render(relay, expires_at, relay.ipv4, stun_or_turn),
-      maybe_render(relay, expires_at, relay.ipv6, stun_or_turn)
+      maybe_render(relay, salt, expires_at, relay.ipv4, stun_or_turn),
+      maybe_render(relay, salt, expires_at, relay.ipv6, stun_or_turn)
     ]
     |> List.flatten()
   end
 
-  defp maybe_render(%Relays.Relay{}, _expires_at, nil, _stun_or_turn), do: []
+  defp maybe_render(%Relays.Relay{}, _salt, _expires_at, nil, _stun_or_turn), do: []
 
   # STUN returns the reflective candidates to the peer and is used for hole-punching;
```

Comment:
```suggestion
  # STUN returns the reflexive candidates to the peer and is used for hole-punching;
```

Reply:
I'll just fix this to remove the outdated code. All credentials are TURN